packages/palette/src/elements/Swiper/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./activeIndex"
+export * from "./percentage"
```

Comment:
Should these be exported from the root of Swiper here?

These are all re-exported in `elements/index` meaning you could do `import { percentage } from '@artsy/palette'` which feels like it isn't the right api.

I could see maybe something like 

```
import { Swiper } from '@artsy/palette'
Swiper.activeIndex
```
That would make a little more sense as it's constrained to the element it's used within instead of polluting the global package scope.

You don't have to follow that particular solution, but I _would_ like to see these not exposed at this level. Of course, open to a counter if I'm missing something.

Reply:
Yeah, I concur. These aren't needed yet so I can cross that bridge when I get to it.